Add a dev page for viewing query plan (#22341)
prestodb/presto: presto-docs/src/main/sphinx/clients/presto-console.rst

.. figure:: ../images/presto_console.png
   :align: center

Plan Viewer
=======================

> Plan Viewer is a tool to generate the graph of a query plan on the Presto Console.

Reviewer: ```suggestion
Plan Viewer generates the graph of a query plan on the Presto Console.
```

> You can access it using this path: ``/ui/dev/index.html`` along with your server IP

Reviewer: ```suggestion
To access the Plan Viewer using a web browser, add the path ``/ui/dev/index.html`` to your server IP
```

> and port number. For example: ``http://localhost:8080/ui/dev/index.html``. When a

Reviewer: ```suggestion
and port number. For example: 

.. code-block:: none
    http://localhost:8080/ui/dev/index.html

```
Suggest changing the formatting of the URI to a code block, like the one earlier on the page, so that a user can copy to Clipboard easier.

Author: fixed

> Presto server restarts, all the query information is reset. However, the

Reviewer: ```suggestion
Use the ``JSON`` tab on the ``Query Details`` page to export and save a query in JSON format. 
```

> information of a query can be exported in JSON format using the ``JSON`` tab on

Reviewer: ```suggestion
```

> the Query Details page and saved locally. Here is where the ``JSON`` tab is located.

Reviewer: ```suggestion
```

.. figure:: ../images/query_details_json.png
   :align: center

> 

Reviewer: ```suggestion

*Note:* When a Presto server restarts, all the query information is reset. 
```
Suggest moving this to here, after you explain how to export the current query.

With the JSON files, you can use the Plan Viewer to view the graph of query plans.

1. Access the Plan Viewer page. For example: ``http://localhost:8080/ui/dev/index.html``

> 2. Click on the ``Browse`` button and select the JSON file you want to examine.

Reviewer: ```suggestion
2. Select the ``Browse`` button and select the JSON file you want to examine. The query plan graph is displayed. 
```
See the [Gitlab documentation style guide recommended word list entry for click](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#click) for discussion and suggestions. 

Moved "The query plan graph is displayed." to here, as it's the result or output of the user's action in step 2.

Author: lesson learned, thanks for the info

> 3. The query plan graph will show up.

Reviewer: ```suggestion
```

> 4. Click on the ``Browse`` button to view other JSON file if needed.

Reviewer: ```suggestion
4. Select the ``Browse`` button to view other JSON files if needed.
```
